builtin/settingtypes.txt

```diff
@@ -711,9 +686,6 @@ fall_bobbing_amount (Fall bobbing factor) float 0.03
 #    Note that the interlaced mode requires shaders to be enabled.
 3d_mode (3D mode) enum none none,anaglyph,interlaced,topbottom,sidebyside,crossview,pageflip
 
-#    Strength of 3D mode parallax.
```

Comment:
This is actually unrelated to shaders, it is for 3D rendering modes.

Reply:
Whoops. Yes. I'll bring that back.

Reply:
Done